### PR TITLE
Removes client commands from server Makefile.

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary
Removes the client commands from the server makefile.

#### Ticket Link
n/a

screen shot 2017-12-11 at 11 29 15 am

### CODE REVIEW

**Comment by coreyhulen** [comment]
I like this.

Are all the appropriate cmds in the webapp repo?  I think we should also update the developer docs before accepting this PR, but otherwise +1

File: Makefile [hunk 1]
```diff
@@ -334,28 +322,21 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 	rm -f config/*.key
 endif
 
-test-server: test-te test-ee
+test: test-te test-ee
```
Comment (crspeller) [inline Makefile]: Jenkins needs the test-server target. Please add that back as an alias.

Comment (ccbrown) [reply Makefile]: Wouldn't be too hard to just have Jenkins try both targets, would it? I think "test" is preferable now that the server is the only thing in the repo.

Comment (crspeller) [reply Makefile]: Ya. That's way better, let me do that.

**Comment by mkraft** [comment]
@crspeller I have now added the docs in https://github.com/mattermost/docs/pull/1662.

**Comment by mkraft** [comment]
Noted in https://github.com/mattermost/mattermost-developer-documentation/pull/65.

**Comment by crspeller** [comment]
Waiting on: https://github.com/mattermost/docs/pull/1662

**Comment by mkraft** [comment]
@crspeller Updated https://github.com/mattermost/docs/pull/1662.

**Pull request closed without merging**